network/bridge_endpointclient_linux.go

```diff
@@ -50,7 +54,8 @@ func NewLinuxBridgeEndpointClient(
 }
 
 func (client *LinuxBridgeEndpointClient) AddEndpoints(epInfo *EndpointInfo) error {
-	if err := epcommon.CreateEndpoint(client.hostVethName, client.containerVethName); err != nil {
+	epc := epcommon.NewEPCommon(client.netlink)
```

Comment:
can we move the epc to the LinuxBridgeEndpointClient struct and reuse in `ConfigureContainerInterfacesAndRoutes` , `SetupContainerInterfaces` etc?